src/mlpack/methods/ann/layer/dropout.hpp

```diff
@@ -47,21 +47,18 @@ namespace ann /** Artificial Neural Network. */ {
  * @tparam OutputDataType Type of the output data (arma::colvec, arma::mat,
  *         arma::sp_mat or arma::cube).
  */
-template <
-    typename InputDataType = arma::mat,
-    typename OutputDataType = arma::mat
->
-class Dropout
-{
+template<
+                typename InputDataType = arma::mat,
```

Comment:
Line 51 and 52, should be tabbed twice from where the previous line began, see https://github.com/mlpack/mlpack/wiki/DesignGuidelines#line-length-and-wrapping for more information.